questionGenerator.py

```diff
@@ -185,6 +186,86 @@ def printTokenCount(self):
             f"Question Generation Token Count: {self.tokenCount}",
         )
 
+    def saveToFile(self):
+        """
+        Save the question data to a file.
+
+        Returns:
+            bool: True if the question data is successfully saved, False otherwise.
+        """
+        saveFolder = os.path.join(outputFolder, self.config.videoToUse)
```

Comment:
Log the `saveFolder` path here for clarity purposes.

Reply:
`questionSavePath` will have the full path. I instead added a log if a folder has been created for the first time to indicate so with the path of the newly created folder.